frontend/src/components/TeledeclarationPreview.vue

```diff
@@ -317,6 +342,7 @@ export default {
       ]
     },
     additionalItems() {
+      if (!this.showAdditionalKeys) return []
```

Comment:
Items and Keys appear to be used interchangeably in this code, which I find confusing. Unless they do refer to different things? (additionalItems vs approKeys vs showAdditionalKeys etc)
Also the context of additional - maybe 'otherMeasures' or 'remainingMeasures' is more explicit